### DISCUSSION TITLE
Ability to Sort Posts by Publication Date

### DISCUSSION THREAD

**Discussion by PiotrSierant** [avitorio/outstatic#230] [General]
Is there a simple way to sort posts by publication date?

![image](https://github.com/avitorio/outstatic/assets/94246053/9da045bf-b78a-48a4-9895-30d6338d24ca)

**Comment by avitorio** [comment]
Hey,

Is your date metadata set as `publishedAt`? 
We sort by date (recent first) for that metadata.

**Comment by PiotrSierant** [comment]
I'm adding a new post (today), and they are clearly being sorted alphabetically. All posts have the metadata publishedAt.

![image](https://github.com/avitorio/outstatic/assets/166378457/101f4b99-541b-48eb-adc5-d0d25926f415)

![image](https://github.com/avitorio/outstatic/assets/166378457/ce7528e3-6ec7-4475-92f9-9a6bd67d0eb8)

**Reply by avitorio** [reply]
Hey @PiotrSierant , I apologize for my previous comment. I just confirmed there's a bug with this. 

I'll create an issue and a fix should be available on our next version.

**Comment by avitorio** [comment]
Hey @PiotrSierant - this is now fixed and released on our latest version `1.4.9`.

Thanks to @amponce for building this feature.
